integreat_cms/cms/templates/languagetreenodes/languagetreenode_form.html

```diff
@@ -21,7 +21,8 @@ <h1 class="heading overflow-hidden text-ellipsis">
             {% if perms.cms.change_languagetreenode %}
                 <div class="flex flex-wrap gap-2">
                     <a href="{% url 'languagetreenodes' region_slug=request.region.slug %}"
-                       class="btn btn-ghost">{% translate "Cancel" %}</a>
+                       class="btn btn-ghost">{% translate
+                    "Cancel" %}</a>
```

Comment:
This made the translation broken and resulted in such appearance of the button

![image](https://github.com/user-attachments/assets/a2048f71-43c5-4a2c-b286-35db1625da61)


```suggestion
                       class="btn btn-ghost">{% translate "Cancel" %}</a>
```